cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/DynamicMapReduceContext.java

```diff
@@ -74,6 +75,7 @@ public DynamicMapReduceContext(Program program,
                                  Arguments runtimeArguments,
                                  MapReduceSpecification spec,
                                  long logicalStartTime, String workflowBatch,
+                                 WorkflowToken workflowToken,
```

Comment:
Hmm, other parameters of this constructor aren't appropriately marked. Maybe the inconsistency would be more confusing. Unsure.